hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketSetPropertyRequest.java

```diff
@@ -239,7 +239,7 @@ public boolean checkQuotaBytesValid(OMMetadataManager metadataManager,
           OMException.ResultCodes.QUOTA_ERROR);
     }
 
-    if (quotaInBytes == 0) {
+    if (quotaInBytes < OzoneConsts.QUOTA_RESET) {
```

Comment:
Seems this should be update to quotaInBytes < OzoneConsts.QUOTA_RESET || quotaInBytes == 0 as well.